runners/flink/job-server/build.gradle

```diff
@@ -45,3 +59,55 @@ runShadow {
   // Enable remote debugging.
   jvmArgs = ["-Xdebug", "-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005"]
 }
+
+class PortableValidatesRunnerConfig {
+  String name
+  String jobServerDriver
+  String jobServerConfig
+  boolean streaming
+}
+
+def createPortableValidatesRunnerTask(Map m) {
```

Comment:
nit: use a lambda and the implicit `it` parameter as it allows you to call the method without any paramemters or with parameters

Reply:
SG.
I am not very sure with the semantics so please take a look again.